docs/run/configure-error-mitigation.mdx

```diff
@@ -265,13 +376,116 @@ from qiskit_ibm_runtime import QiskitRuntimeService, Sampler, Options
 service = QiskitRuntimeService()
 options = Options()
 options.resilience_level = 1
-options.optimization_level = 3
+options.optimization_level = 1
 backend = service.backend("ibmq_qasm_simulator")
 
 sampler = Sampler(backend, options=options)
 ```
 
-## Advanced resilience options
+<span id="advanced-error"></span>
+## Custom error settings
+
+With Estimator V2, you can turn on and off individual error mitigation and suppression methods, including dynamical decoupling, gate and measurement twirling, measurement error mitigation, and ZNE.
+
+<span id="advanced-options"></span>
+## Custom error settings (V2 primitives)
+With the V2 primitives, you can turn on and off individual error mitigation and suppression methods. {/*See the API reference for the most up-to-date information on available options.*/}
+
+**Available options:**
+
+{/*Need to verify which options actually make it into Sampler.  Possibly some of the last 5 will.*/}
+
+| Category   | Option                   | Description                                                                                                                                                                                                                             | Default        | Notes          |
```

Comment:
I deleted the whole table since we deleted it in the migration guide; should I put it back in with just options related to error suppression / mitigation?